addons/l10n_vn_edi_viettel/tests/test_edi.py

```diff
@@ -49,7 +49,7 @@ def setUpClass(cls):
             'street': '3 Alley 45 Phan Dinh Phung, Quan Thanh Ward',
             'state_id': cls.env.ref('base.state_vn_VN-HN').id,
             'country_id': cls.env.ref('base.vn').id,
-            'vat': '0100109106-506',
+            'vat': '0100109106506', # TODO: check in base_vat, ...
```

Comment:
@vin-odoo Any idea if everyone wants this - or not?  (stdnum seems to compact without)

Reply:
I checked with our Vietnamese expert and got the reply:

> Hi Nicolas, the VAT in Vietnam actually doesn't have -
It is only numbers

So this change is correct :ok_hand: I could see resources online displaying the - but I guess it's not common practice so removing it should never be an issue.

Reply:
Mostly if it is in format, it is optional and more of a beautification and we prefer the short form, but sometimes the - is really in handy.  We can override it to use the format...

Reply:
adapted like you suggested